### DISCUSSION TITLE
Is the store ID used by whatsapp?

### DISCUSSION THREAD

**Discussion by luis-guideti** [Auties00/WhatsappWeb4j#214] [Q&A] [answered]
I noticed the random store id goes up to 16k, is this number used by whatsapp or is for the api only? Im asking because my usage here generate ids up to Integer.MAX_VALUE, I am having some issues that may not be related at all with this number so I wonder if there is an actual up/lower limit

**Comment by Auties00** [comment, marked as answer]
It's used by whatsapp. The upper limit was determined when I reversed the web client, i cannot remember rn if having it higher can create problems

**Reply by luis-guideti** [reply]
to high of a number appears to cause it to not receive the initial sync

**Reply by Auties00** [reply]
Oh that's interesting
